cmd/k8s-operator/sts.go

```diff
@@ -86,7 +86,6 @@ const (
 	// ensure that it does not get removed when a ProxyClass configuration
 	// is applied.
 	podAnnotationLastSetClusterIP         = "tailscale.com/operator-last-set-cluster-ip"
-	podAnnotationLastSetHostname          = "tailscale.com/operator-last-set-hostname"
```

Comment:
Hostname is now in the configfile and proxy pods will be restarted when hostname changes by `podAnnotationLastSetConfigFileHash` annotation change